pkg/operator/starter.go

```diff
@@ -126,9 +133,17 @@ func RunOperator(ctx context.Context, kubeconfig *restclient.Config) error {
 		configInformers.Config().V1().Images(),
 	)
 
+	var recorder events.Recorder
+	controllerRef, err := events.GetControllerReferenceForCurrentPod(context.TODO(), kubeClient, defaults.ImageRegistryOperatorNamespace, nil)
```

Comment:
check to be sure you have the pod name envvar set: Something like https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/manifests/0000_20_kube-apiserver-operator_06_deployment.yaml#L70-L73